### ISSUE TITLE
Error In Adding IP Camera

### ISSUE THREAD

**Issue by korrapati-kris** [ant-media/Ant-Media-Server#373]
# Issue
When I try to add IP camera, I am getting message "New broadcast is not created".

### Short description
After logging into the main console, and choosing "IP Camera" as source and when I try to add the IP Camera, its throwing an error "New Broadcast is Not Created".

The configurations that I tried to add the camera.
Conf 1:
Name : Camera
Onvif IP URL : rtsp://<myip>:554/cam/realmonitor?channel=1&subtype=0&unicast=true&proto=Onvif
User Name : user_name
Password : password

Conf 2:
Name : Camera
Onvif IP URL : <myip>:554
User Name : user_name
Password : password

Conf 3:
Name : Camera
Onvif IP URL : rtsp://<Username>:<Password>@<myip>:<Port>/cam/realmonitor?channel=1&subtype=0
User Name : user_name
Password : password

TCP Port
36000
UDP Port
36001
HTTP Port
80
RTSP Port
554
HTTPS Port
443
RTSP Format
rtsp://<Username>:<Password>@<IP Address>:<Port>/cam/realmonitor?channel=1&subtype=0
channel: Channel1-1; subtype: Code-Stream Type, Main Stream 0, Sub Stream 1.

### Environment

[] Operating system and version: Ubuntu
I am using the community edition.

I tried with many configurations but none got succeeded. Any help would be appreciated. Willing to give the remote access if needed.

Thank you.

**Comment by davutcavdar** [comment]
Hi @korrapati-kris 

Please make sure that, if you add IP Camera directly ie New Live Stream-> IP Camera, you need to write Onvif Url such as IPAddr:OnvifPort not RTSP URL, on the other hand, if you have RTSP URL of IP Camera, please use New Live Stream -> Stream Source and use this format rtsp://[username]:[password]@<IP Addr>:<port>/<other parameters>

**Comment by phoenixyork** [comment]
Yes i tried IPAddr:OnvifPort but still it says "New Broadcast is Not Created" -Any help would be appreciated

**Comment by davutcavdar** [comment]
There should be some logs created after your tries. 

If you use Ant Media Server on local please follow terminal and get the related log rows if you use on remote, please get it from root_path->log->ant-media-server.log. 
 
You can either share them here or send to contact [at] antmedia.io. We will write comments here for anyone who is facing the same issue.

**Comment by thangamdus** [comment]
Hi i dont think my Panansonic HC V785 is ONVIF enabled. can i still input my camera's local ip  - because it brings the message New broadcast is not created. Please help to set up.

**Comment by davutcavdar** [comment]
@thangamdus  If your camera has RTSP support and you know its RTSP address, you can fetch streams from it by adding it as stream source.
